packages/create-workleap/src/bin.ts

```diff
@@ -1,111 +1,107 @@
 #!/usr/bin/env node
-import * as child_process from "child_process";
-import process from "process";
-import { spinner } from "@clack/prompts";
-
-import { intro, select, note, text, outro } from "./prompts";
-import * as pkg from "../package.json";
+import process from "node:process";
+import * as p from "@clack/prompts";
+import fs from "node:fs";
+import path from "node:path";
+import colors from "picocolors";
+import { generateProject } from "./generateProject.js";
+import packageJson from "../package.json" assert { type: "json" };
+import { type TemplateId } from "./templates.js";
+
+let outputDir = process.argv[2];
+
+p.intro(colors.gray(`${packageJson.name} - v${packageJson.version}`));
+
+// Ask for output directory
+if (!outputDir) {
+    const dir = await p.text({
+        message: "Where should we create your project?",
+        placeholder: "  (hit Enter to use current directory)"
+    });
 
-const DefaultOutputDirectory = "./my-new-project";
-const NameParameterPosition = 2; // TODO validate position of parameter once ask with `pnpm create`
+    if (p.isCancel(dir)) { process.exit(1); }
```

Comment:
Aren't our ESLint rules forbidding to have both braces on the same line?

Reply:
Nope. For cases like this, guard statement, it's convenient to be able to add it as one line